vermouth/processors/canonicalize_modifications.py

```diff
@@ -1,15 +1,36 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-from .processor import Processor
 
 from collections import defaultdict
 import itertools
 
 import networkx as nx
 
+from .processor import Processor
+
 
 class PTMGraphMatcher(nx.isomorphism.GraphMatcher):
+    '''
```

Comment:
Ohhh. This is cheating pylint! Shame on you :stuck_out_tongue:

Reply:
I thought this was to keep sphinx from nitpicking and blowing up, but I can't reproduce it at the moment.